connect/src/com/telenor/connect/ConnectSdk.java

```diff
@@ -145,20 +179,23 @@ public static Fragment getAuthFragment(Map<String, String> parameters) {
     /**
      * Get a valid Access Token. If a non-expired one is available, that will be given to the
      * callback {@code onSuccess(String accessToken)} method.
-     *
+     * <p>
      * If it is expired, it will be refreshed and then returned. This requires a network call.
      *
      * @param callback callback that will be called on success or failure to update.
      * @throws ConnectRefreshTokenMissingException if no Request Token is available
      */
     public static synchronized void getValidAccessToken(AccessTokenCallback callback) {
         Validator.sdkInitialized();
-        sConnectIdService.getValidAccessToken(callback);
+        sdkProfile.getConnectIdService().getValidAccessToken(callback);
     }
 
     public static synchronized String getAccessToken() {
         Validator.sdkInitialized();
-        return sConnectIdService.getAccessToken();
+        if (sdkProfile.getConnectIdService() != null) {
```

Comment:
MC is doing lazy initialisation (not before the first authenticate request), so this is necessary in case the app is trying to get tokens without starting the auth flow first. Does not apply to Connect Id, though, since it is initialised statically.